### PR TITLE
add missing roles

### PR DESCRIPTION
**Pull request by jharrod**
Adds missing roles

### CODE REVIEW

File: deploy/astra-connector/clusterrole.yaml [hunk 1]
```diff
@@ -19,5 +19,5 @@ rules:
   resources: ["tridentversions", "tridentorchestrators"]
   verbs: ["watch", "list", "get"]
 - apiGroups: ["astra.netapp.io"]
-  resources: ["applications", "appmirrorrelationships", "appmirrorupdates", "appvaults", "autosupportbundles", "backups", "backupinplacerestores", "backuprestores", "exechooks", "exechooksruns", "pvccopies", "pvcerases", "resourcebackups", "resourcedeletes", "resourcerestores", "resourcesummaryuploads", "resticvolumebackups", "resticvolumerestores", "schedules", "snapshotinplacerestores", "snapshotrestores", "snapshots", "astraconnectors"]
-  verbs: ["watch", "list", "get"]
\ No newline at end of file
+  resources: ["applications", "appmirrorrelationships", "appmirrorupdates", "appvaults", "autosupportbundles", "backups", "backupinplacerestores", "backuprestores", "exechooks", "exechooksruns", "pvccopies", "pvcerases", "resourcebackups", "resourcedeletes", "resourcerestores", "resourcesummaryuploads", "resticvolumebackups", "resticvolumerestores", "schedules", "snapshotinplacerestores", "snapshotrestores", "snapshots", "astraconnectors", "shutdownsnapshots", "autosupportbundleschedules", "kopiavolumebackups"]
```
Comment (vivintw) [inline deploy/astra-connector/clusterrole.yaml:22]: what about kopiavolumerestores ?

File: deploy/astra-connector/role.yaml [hunk 2]
```diff
@@ -32,6 +32,8 @@ rules:
     "snapshots",
     "snapshotinplacerestores",
     "snapshotrestores",
-    "astraconnectors"
+    "astraconnectors",
+    "autosupportbundleschedules",
+    "kopiavolumebackups"
```
Comment (palicena) [inline deploy/astra-connector/role.yaml:37]: checking just in case; only adding `shutdownsnapshots` as a clusterrole (above), not as a role here?

**Review by palicena — APPROVED** [review]

**Review by oscarr-ntap — APPROVED** [review]

File: deploy/astra-connector/clusterrole.yaml (hunk 1 above)
Comment (jharrod) [reply deploy/astra-connector/clusterrole.yaml:22]: didn't know that was a thing. I can add it

File: deploy/astra-connector/role.yaml (hunk 2 above)
Comment (jharrod) [reply deploy/astra-connector/role.yaml]: thanks for checking. That one was previously added. You might have to expand the code view to see it

**Comment by jharrod** [comment]
Closing this. I wasn't aware that these files do get automatically updated via cicd when changed in connector repo. 
I will I need to add 'kopiavolumerestores' to connector repo and it will be reflected here

**Pull request closed without merging**